packages/agent/src/components/json-components/raw-json-view.tsx

```
@@ -0,0 +1,66 @@
+              <CheckIcon className="h-4 w-4 text-green-500" />
+            </motion.div>
+          ) : (
+            <CopyIcon className="h-4 w-4" />
+          )}
+        </Button>
+      </motion.div>
+
+      <div className="bg-gray-50 dark:bg-gray-900 p-5 rounded-xl overflow-auto max-h-[500px] text-sm font-mono shadow-inner border border-purple-100 dark:border-purple-900/50">
+        <pre
+          className="language-json"
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)